weblate/addons/models.py

```diff
@@ -231,6 +231,13 @@ def handle_addon_event(
                     else:
                         # Callback is used in tasks
                         method(addon)
+                    AddonActivityLog.objects.create(
+                        addon=addon,
+                        component=component,
+                        event=event.label,
+                        # TODO: What to add here?
+                        details={},
+                    )
```

Comment:
* This should definitely be out of try/except block.
* As for the content, I think the best approach would be if the `method` would return the content.
* There should be a way to avoid logging for some frequently triggered add-on events like EVENT_UNIT_* or EVENT_STORE_*.
* In case of error (see except clause below), the error should be stored.